build/packages.targets

```diff
@@ -1,21 +1,18 @@
 <Project>
     <PropertyGroup>
-        <MicrosoftBuildPackageVersion Condition="'$(MicrosoftBuildPackageVersion)' == ''">16.5.0-preview-19606-01</MicrosoftBuildPackageVersion>
+        <MicrosoftBuildPackageVersion Condition="'$(MicrosoftBuildPackageVersion)' == ''">16.6.0</MicrosoftBuildPackageVersion>
         <NewtonsoftJsonPackageVersion Condition="$(NewtonsoftJsonPackageVersion) == ''">9.0.1</NewtonsoftJsonPackageVersion>
         <SystemPackagesVersion>4.3.0</SystemPackagesVersion>
-        <VSComponentsVersion>16.4.280</VSComponentsVersion>
+        <VSComponentsVersion>16.6.255</VSComponentsVersion>
         <VSFrameworkVersion>16.6.30107.105</VSFrameworkVersion>
         <VSServicesVersion>16.153.0</VSServicesVersion>
-        <VSThreadingVersion>16.6.13</VSThreadingVersion>
+        <VSThreadingVersion>16.7.54</VSThreadingVersion>
```

Comment:
Where do you get this exact number version?

Reply:
I looked for latest on nuget.org